api.ts

```diff
@@ -1699,11 +1739,41 @@ export interface SyncTableOptions<
    * sync tables that have a dynamic schema.
    */
   dynamicOptions?: DynamicOptions;
+
   /**
-   * A function to autocomplete values for properties marked with `mutable` and `autocomplete`.
+   * An autocomplete function to use for any dynamic schema properties.
+   * The name of the property that's being modified by the doc editor
+   * is available in autocomplete function's context parameter.
```

Comment:
Nit, should it be "in _the_ autocomplete function's"?

Reply:
Done